messaging/src/main/java/org/frankframework/jms/JmsMessagingSourceFactory.java

```diff
@@ -65,8 +65,8 @@ public synchronized MessagingSource getMessagingSource(String connectionFactoryN
 	private Context getContext() throws IbisException {
 		try {
 			return new InitialContext();
-		} catch (Throwable t) {
-			throw new IbisException("could not obtain context", t);
+		} catch (Exception e) {
+			throw new IbisException("could not obtain context", e);
```

Comment:
Ik denk niet dat dit handig is....
```suggestion
		} catch (Throwable t) {
			throw new IbisException("could not obtain context", t);
```

Reply:
Ben ik het mee eens, je veranderd de semantiek. Er werd waarschijnlijk met goede reden een `Throwable` afgevangen ipv een `Exception`.

Reply:
Wist ik niet. Aangepast met een annotatie erboven, dan zien we hem de volgende keer niet weer.